stock_warehouse_orderpoint_stock_info/models/stock_warehouse_orderpoint.py

```diff
@@ -14,14 +14,18 @@ class StockWarehouseOrderpoint(models.Model):
 
     @api.multi
     def _compute_product_available_qty(self):
-        for rec in self:
-            product_available = rec.product_id.with_context(
-                location=rec.location_id.id
-                )._product_available()[rec.product_id.id]
-            rec.product_location_qty = product_available['qty_available']
-            rec.incoming_location_qty = product_available['incoming_qty']
-            rec.outgoing_location_qty = product_available['outgoing_qty']
-            rec.virtual_location_qty = product_available['virtual_available']
+        for loc in self.mapped('location_id'):
+            order_in_loc = self.filtered(lambda x: x.location_id == loc)
+            products = order_in_loc.mapped('product_id').with_context(
+                location=loc.id
+                )._product_available()
```

Comment:
@fmdl You should call `_compute_quantities_dict`in place of `_product_available`. `_product_available`is kept into Odoo for compatibility. https://github.com/odoo/odoo/blob/10.0/addons/stock/models/product.py#L110